benchmarks/bench_micro.c

```diff
@@ -101,18 +208,18 @@ main(int argc, char *argv[])
 
 	/* default values of parameters */
 	unsigned n_threads = 10;
-	unsigned write_count = 100000;
+	unsigned ops_count = 100000;
 	size_t cache_max_size = VMEMCACHE_MIN_POOL;
 	size_t cache_fragment_size = VMEMCACHE_MIN_FRAG;
 	unsigned nbuffs = 10;
-	size_t min_size = 8;
-	size_t max_size = 64;
+	size_t min_size = 128;
+	size_t max_size = 256;
```

Comment:
This doesn't look like "only a refactor". Such changes needs to be in a separate commit.

Reply:
Removed